lua/nvim-treesitter/textobjects/shared.lua

```diff
@@ -13,7 +13,7 @@ function M.available_textobjects(lang)
   return found_textobjects
 end
 
-function M.textobject_at_point(query_string, pos, bufnr)
+function M.textobject_at_point(query_string, pos, bufnr, lookahead)
```

Comment:
Just in case that there will be more customizations in the futures... Could you put lookahead in a options/opts parameter?